spec/access-identify.md

```diff
@@ -0,0 +1,303 @@
+## `access/identify`
+
+Upload protocol defines [access/identify][] capability which is very flexible yet powerful so it can be very confusing. This document attempts to wolk through some user stories to try and clarify.
+
+#### identify with email
+
+"Accounting" service MAY delegate other "verifier" service(s) to do out of bound user verification e.g. sign-in via email link style flow.
+
+To do so _accounting_ must delegate a capability to _verifier_ service which will allows it:
+
+1. To create an accounts under `did:email:` namespace.
+2. To associate other DIDs with accounts under `did:email:` namespace.
+
+:::danger
+It is important to call out that accounting needs to put a lot of trust into such service, as malicious verifier could get control of other accounts
+:::
+
+```js
+{
+  issuer: "did:key:zAccount",
+  audience: "did:key:zVerify",
+  capabilities: [
+    {
+      can: "access/identify",
+      with: "did:email:*"
+    }
+  ]
+}
+```
+
+Now if Alice goes to web3.app on first load app will generate local keypair and encode public key as `did:key:zAlice` DID. web3.app supports email login through `did:key:zVerify` service so when Alice enters her email address web3.app will send request to `did:key:zVerify`
+
+```js
+{
```

Comment:
```suggestion
```js
// web3.app signs this with Alice's key and sends it to the verifier service
{
```